filters/contains-non-ascii/filter.py

```diff
@@ -0,0 +1,15 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class TextContainsNonAsciiFilter(SentenceOperation):
+    tasks = [e for e in TaskType]
+    languages = ["en", "de", "all"]
```

Comment:
@kaustubhdhole correct me if I am wrong, but if `all` is set, the other two do not have to, right?

Reply:
Yes, "all" represents all languages and the others need not be added.